apps/test/unit/templates/rubrics/EvidenceLevelsTest.jsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import {expect} from '../../../util/reconfiguredChai';
 import {shallow} from 'enzyme';
 import EvidenceLevels from '@cdo/apps/templates/rubrics/EvidenceLevels';
+import {UNDERSTANDING_LEVEL_STRINGS} from '@cdo/apps/templates/rubrics/rubricHelpers';
 
 const DEFAULT_PROPS = {
   evidenceLevels: [
-    {id: 1, understanding: 1, teacherDescription: 'test'},
-    {id: 2, understanding: 2, teacherDescription: 'test'},
+    {id: 1, understanding: 1, teacherDescription: 'test1'},
+    {id: 2, understanding: 2, teacherDescription: 'test2'},
```

Comment:
I changed these just so they would be a bit more different from one another.   Right now we don't use `teacherDescription`, but in the future we might!

Reply:
hm we don't? that feels like a miss 😬

Reply:
I updated these tests to include some coverage of the teacherDescription.

Reply:
Yay thank you!